src/routes/index.ts

```diff
@@ -0,0 +1,14 @@
+import express from "express";
+
+import defaultRoute from "./default";
+import emailAuthDataRoute from "./emailAuthData";
+import lockRoute from "./lock";
+import metadataRoute from "./metadata";
+
+const router = express.Router();
+router.use("/", defaultRoute);
+router.use("/", emailAuthDataRoute);
```

Comment:
route was different previously for emailAuthDataRoute, it was /auth

Reply:
fixed